cclib/io/moldenwriter.py

```diff
@@ -197,6 +202,54 @@ def _mo_from_ccdata(self):
 
         return lines
 
+    def _freq_from_ccdata(self):
+
+        lines = []
+
+        if hasattr(self.ccdata, "vibfreqs"):
+            vibfreqs = self.ccdata.vibfreqs
+            vibfreqs_lines = ["[FREQ]"]
+            vibfreqs_lines.extend([f"{vibfreq:16.8f}" for vibfreq in vibfreqs])
+            lines.append("\n".join(vibfreqs_lines))
+
+        if hasattr(self.ccdata, "atomcoords") and hasattr(self.ccdata, "atomnos"):
+            atomcoords = utils.convertor(self.ccdata.atomcoords[0], "Angstrom", "bohr")
+            atomsyms = (self.pt.element[atomno] for atomno in self.ccdata.atomnos)
+            atomcoords_lines = ["[FR-COORD]"]
+            for atomsym, atomcoord in zip(atomsyms, atomcoords):
+                atomcoords_lines.append(
+                    "{:3s} {:15.8f} {:15.8f} {:15.8f}".format(atomsym, *atomcoord)
+                )
+            lines.append("\n".join(atomcoords_lines))
+
+        if hasattr(self.ccdata, "vibdisps"):
+            vibdisps = self.ccdata.vibdisps
```

Comment:
do these not need converted to delta Bohr?

Reply:
They do, I missed it.